polychrom/hdf5_format.py

```diff
@@ -127,42 +130,161 @@ class HDF5Reporter(object):
+        
+        Parameters
+        ----------
+        
+        folder : str 
+            Folder to save data to. 
+            
+        max_data_length: int, optional (default=50)
+            Will save data in groups of max_data_length blocks 
+            
+        overwrite: bool, optional (default=False)
+            Overwrite an existing trajectory in a folder. 
+        
+        check_exists: bool (optional, default=True)
+            Raise an error if previous trajectory exists in the folder 
+            
+        blocks_only: bool, optional (default=False) 
+            Only save blocks, do not save any other information 
+        
+
         
         """
-        prefixes = ["blocks", "applied_forces", "initArgs", "starting_conformation", "energy_minimization"]
+        
+        self.prefixes = ["blocks", "applied_forces", "initArgs", "starting_conformation", "energy_minimization", 
+                        "forcekit_polymer_chains"]  # these are used for inferring if a file belongs to a trajectory or not 
+        self.counter = {}  # initializing all the options and dictionaries 
+        self.datas = {}
+        self.max_data_length = max_data_length
+        self.h5py_dset_opts = h5py_dset_opts
+        self.folder = folder
+        self.blocks_only = blocks_only
+                                
+        
         if not os.path.exists(folder):
             os.mkdir(folder)
-            
+
         if overwrite: 
             for the_file in os.listdir(folder):
                 file_path = os.path.join(folder, the_file)
                 if os.path.isfile(file_path):                    
-                    for prefix in prefixes:
+                    for prefix in self.prefixes:
                         if the_file.startswith(prefix):
                             os.remove(file_path)
                 else:
                     raise IOError("Subfolder in traj folder; not deleting. Ensure folder is correct and delete manually. ")
-                        
-                    
-        if len(os.listdir(folder)) != 0:
-            for the_file in os.listdir(folder):            
-                for prefix in prefixes:
-                    if the_file.startswith(prefix):
-                        raise RuntimeError(f"folder {folder} is not empty")
-        self.counter = {}
-        self.datas = {}
-        self.max_data_length = max_data_length
-        self.h5py_dset_opts = h5py_dset_opts
-        self.folder = folder
-        self.blocks_only = blocks_only
+
+
+        if check_exists:
+            if len(os.listdir(folder)) != 0:
+                for the_file in os.listdir(folder):            
+                    for prefix in self.prefixes:
+                        if the_file.startswith(prefix):
+                            raise RuntimeError(f"folder {folder} is not empty: set check_exists=False to ignore")
+
+    def continue_sim(self, continue_from=None, 
```

Comment:
The name didn't make sense to me until i read the code... this is almost analogous to a file seek, no? Maybe recorders should mimic file-like objects (`seek`, `tell`, `truncate`) but work with blocks instead of lines.